sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -80,10 +80,8 @@ class HiveContext private[hive](
   protected[sql] override def sharedState: HiveSharedState = {
     sparkSession.sharedState.asInstanceOf[HiveSharedState]
   }
-
```

Comment:
It was because I touched the file (not the line) and while I was rebasing I noticed the offending line. When I was at it I decided to remove it since I was fixing the merge conflict.